### ISSUE TITLE
Moving to GitHub has broken the staging scripts for the Core-Model

### ISSUE THREAD

**Issue by lbross** [LANDIS-II-Foundation/Core-Model#4]
Instead of storing .prj and .sln files in source control, the Core-Model relies on a series of .lua scripts, .bat and .txt files to download the current third-party libraries and generate the .prj and .sln files. These scripts have google-code repository urls hard-coded into them. Since the source code has been moved, the scripts no longer work and will only become less usable when google code goes offline.

Unless we have a resource to update these scripts, we may want to store the libraries with the core-model project and reference them directly. The library version have been static thus far. We can also store .sln and .prj files to avoid having to use the scripts.

Also open to other suggestions ...

**Comment by lbross** [comment]
I found that the scripts were broken due to a folder that was not migrated from SVN to GitHub. I added this folder to GitHub and the scripts work. This is a temporary solution as the LSML downloads still reside on GoogleCode. GitHub doesn't offer a distribution mechanism similar to GoogleCode downloads. GoogleCode recommends moving the downloads to Google Drive which could work but there is still work to be done. The scripts have some checksum and security elements that don't appear to be compatible with Google Drive. I'm not a fan of moving the downloads into source control because the repository is already quite large and syncs take a while.
